### DISCUSSION TITLE
Workflows with Python

### DISCUSSION THREAD

**Discussion by NeoZ16** [Vector35/binaryninja-api#4107] [API] [answered]
The [example](https://api.binary.ninja/binaryninja.workflow-module.html) for worklows include a small python activity.
However the analysis_context object you that gets passed into the function is a C-object and it is not really possible to access the C++ Functions (e.g. "context->GetFunction()") .
Is there any way to do this or is this just not implemented yet?

**Comment by bpotchik** [comment, marked as answer]
This is not recommended due to the performance of python and multi-threading. However, there is some utility with regards to creating a proof of concept transform. Here is a good python example that's in the wild: https://publish.obsidian.md/binja-vm/Workflow 

Better support for python workflows is forthcoming.
